lib/matplotlib/font_manager.py

```diff
@@ -275,7 +270,7 @@ def findSystemFonts(fontpaths=None, fontext='ttf'):
     if fontpaths is None:
         if sys.platform == 'win32':
             installed_fonts = _get_win32_installed_fonts()
-            fontpaths = MSUserFontDirectories + [win32FontDirectory()]
+            fontpaths = []
```

Comment:
```suggestion
            fontpaths = MSUserFontDirectories
```

Might work, but seems against the spirit of this change....